src/css/blog-post.module.scss

```diff
@@ -310,9 +318,17 @@
 // Responsive Design
 
 @media screen and (max-width: 768px) {
```

Comment:
The pixels for the media screen also goes to the `variables.scss` under a `breakpoints` list.
Here's an example of how it would look like on the `variables.scss` file:
```
$breakpoints: (
  sm: 940px,
  xs: 768px,
);
```

And then you can call this line like this:
`@media screen and (max-width: map-get($map: $breakpoints, $key: xs)) {`
This way if a breakpoint changes, you only have to change it one place 😄

Reply:
Done